### PR TITLE
Add small section about what happens to Gists if you change username.

### LINKED ISSUE #20769: Add information about what happens with Gists after a username change
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username

### What part(s) of the article would you like to see updated?

Add a small section explaining the impact on Gists (PR incoming for this)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by billythekid**
### Why:

Closes #20769

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Adding a small section about Gists and how they're affected if you change username.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by cmwilson21** [comment]
@billythekid Thank you for opening a PR and linking it to your issue! ✨

**Review by PIYUSH-MISHRA-00 — COMMENTED** [review]
An explanatory section about what to Gists if you change username.

**Review by 4bambi — COMMENTED** [review]
I suppose I'm learning

**Review by vgrl — COMMENTED** [review]
Thanks @billythekid ⚡ , I'm going to make a tiny change and then I'll approve and merge this.

File: content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ After changing your username, links to your previous profile page, such as `http
 
 {% ifversion fpt or ghec %}Git commits that were associated with your {% data variables.product.product_name %}-provided `noreply` email address won't be attributed to your new username and won't appear in your contributions graph.{% endif %} If your Git commits are associated with another email address you've [added to your GitHub account](/articles/adding-an-email-address-to-your-github-account), {% ifversion fpt or ghec %}including the ID-based {% data variables.product.product_name %}-provided `noreply` email address, {% endif %}they'll continue to be attributed to you and appear in your contributions graph after you've changed your username. For more information on setting your email address, see "[Setting your commit email address](/articles/setting-your-commit-email-address)."
 
+## Your Gists
+
+After changing your username, the URLs to any public or secret Gists will also change and previous links to these will return a 404 error. We recommend updating the links to these Gists anywhere you may have shared them.
```
Comment (vgrl) [inline content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username.md:78]: Tiny change for consistency with how we capitalise gist elsewhere (for example, https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists).

```suggestion
## Your gists

After changing your username, the URLs to any public or secret gists will also change and previous links to these will return a 404 error. We recommend updating the links to these gists anywhere you may have shared them.
```

**Review by vgrl — DISMISSED** [review]
Update Gists to gists

**Review by vgrl — APPROVED** [review]
Looks good on [preview](https://docs-20770-c961e4.preview.ghdocs.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username#your-gists) 🎉

**Pull request merged**